windows/freeze.py

```diff
@@ -174,7 +174,7 @@ def copy_plugins(name):
  Flags: postinstall nowait skipifsilent;
 
 '''.format(
-    version=appinfo.version,
+    version=appinfo.running_version(),
```

Comment:
I think that the same comments as for macosx/... files hold also here: this version is used in the compiled Windows installer for the release, which is probably built from the Git repository.